readme.md

```diff
@@ -4,11 +4,11 @@
 
 > It's like [Vivus](https://maxwellito.github.io/vivus/) but for a disposable usage. Powered by CSS animations.
 
-Single page app to create independent stroke animated SVGs.
+Single page app to create independent stroke animated SVGs. Open your SVG to make sure that the < g > tag has stroke in there somewhere.
```

Comment:
I understand your point but it's more complex than that. It's not related to `<g>` tags only but to any compatible element like 'path', 'line', 'ellipse', 'circle', 'polygon', 'polyline' and 'rect'. Maybe mentioning something like "ensure your SVG elements contains `stroke` properties and `fill=none`"